app/models/course.rb

```diff
@@ -516,11 +516,20 @@ def program_type_description
   def description
     return qualifications_description if teacher_degree_apprenticeship?
 
-    study_mode_string = (full_time_or_part_time? ? ', ' : ' ') +
-                        study_mode_description
     qualifications_description + study_mode_string + program_type_description
   end
```

Comment:
I know the summary above is almost a duplication of this description but my plan is to remove this description field after the feature flag is removed.